### PR TITLE
Update for Earlybird usage

### PR DESCRIPTION
**Pull request by FungOliver**
This PR request:
-  fixes the type in the launch.json file 
- updates the lang dune from 3.6 to 3.7 to allow breakpoints to be hit (https://github.com/hackwaly/ocamlearlybird#troubleshooting)
- updates the opam.locked accordingly

### CODE REVIEW

File: goblint.opam.locked [hunk 1]
```diff
@@ -50,12 +50,12 @@ depends: [
   "cpu" {= "2.0.0"}
   "csexp" {= "1.5.1"}
   "ctypes" {= "0.20.1"}
-  "dune" {= "3.6.1"}
-  "dune-build-info" {= "3.6.1"}
-  "dune-configurator" {= "3.6.1"}
-  "dune-private-libs" {= "3.6.1"}
-  "dune-site" {= "3.6.1"}
-  "dyn" {= "3.6.1"}
+  "dune" {= "3.7.1"}
```
Comment (sim642) [inline goblint.opam.locked:53]: The CI fails because some more dune dependencies need their versions bumped.

Comment (FungOliver) [reply goblint.opam.locked:53]: Should now be fixed in [85e6fdb](https://github.com/goblint/analyzer/pull/1251/commits/85e6fdbfcba3d42bf35b44f1c41171e958890451)

File: dune-project [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(lang dune 3.6)
```
[the remark this answers is not in the document]
Comment (FungOliver) [reply dune-project:1]: I rebuild the file and added this to [85e6fdb](https://github.com/goblint/analyzer/pull/1251/commits/85e6fdbfcba3d42bf35b44f1c41171e958890451)

**Comment by FungOliver** [comment]
It seems like Gobview downgrades the dune dependency back down to 3.6.1. I created a corresponding PR to fix this issue https://github.com/goblint/gobview/pull/34

**Pull request merged**